### PR TITLE
Added code for login/registration requirement

### LINKED ISSUE #32: If user tries to create an account with an existing username, program crashes

### PR DESCRIPTION
**Pull request by tbuwadi**
Hi, please review my code for the login registration requirement

### CODE REVIEW

**Review by John-Merritt — APPROVED** [review]
Looks good

File: HotelBooking/Views/registration_view.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        self.initiate_options()
+        self.authentication_controller = AuthenticationController()
+
+    def show(self):
+        big_print("Registration PORTAL")
+        self.create_customer()
+
+    def create_customer(self):
+        customer_id = self.prompt_and_get_answer(PROMPT_KEY['USERNAME'])
+        isDuplicateUsername = self.authentication_controller.customer_exists(customer_id)
```
Comment (mclhtay) [inline HotelBooking/Views/registration_view.py]: pascal case please.

File: HotelBooking/Database/create_database.sql [hunk 2]
```diff
@@ -1,7 +1,11 @@
 CREATE TABLE customer
 (
   customer_id VARCHAR(10) NOT NULL PRIMARY KEY,
-  customer_password VARCHAR(50) NOT NULL
+  customer_password VARCHAR(50) NOT NULL,
+  customer_name VARCHAR(100),
+  customer_address VARCHAR(120),
+  customer_cellnumber INTEGER,
```
Comment (mclhtay) [inline HotelBooking/Database/create_database.sql]: why not `customer_cell_number` and `customer_credit_card_number` instead?

**Review by mclhtay — CHANGES_REQUESTED** [review]
Please add unit tests for models and controllers.

**Comment by tbuwadi** [comment]
@mclhtay pls see new commits with requested changes!

**Review by mclhtay — APPROVED** [review]
Approve on the basis that the author did manual UI testing.

**Pull request merged**